docs/compute-element/install-htcondor-ce.md

```diff
@@ -29,8 +29,11 @@ Before starting the installation process, consider the following points, consult
     `gratia`
     You will also need to create Unix accounts for each collaboration that you wish to support.
     See details in the ['Configuring authentication' section below](#configuring-authentication).
--   **SSL certificate:** The HTCondor-CE service uses a host certificate at `/etc/grid-security/hostcert.pem` and an
-    accompanying key at `/etc/grid-security/hostkey.pem`
+-   **SSL certificate:** The HTCondor-CE service uses a host certificate named
+    `hostcert.pem` and an accompanying key named `hostkey.pem`.
+    -  If using a Let's Encrypt cert, place the cert and key into `/etc/pki/tls`
```

Comment:
Specifically, the cert/key should match the default HTCondor-CE (and HTCondor) values:

```
AUTH_SSL_SERVER_CERTFILE = /etc/pki/tls/certs/localhost.crt
AUTH_SSL_SERVER_KEYFILE = /etc/pki/tls/private/localhost.key
```

Reply:
Ah, tricky; updated again.